Dynavity/Dynavity/view/CanvasElementMapView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CanvasElementMapView.swift
+//  Dynavity
+//
+//  Created by Hans Sebastian Tirtaputra on 15/3/21.
+//
+
+import SwiftUI
+
+struct CanvasElementMapView: View {
+    @Binding var elements: [CanvasElementProtocol]
+
+    var body: some View {
+        ZStack {
+            ForEach(elements, id: \.visualID) { element in
```

Comment:
Building onto @sebastiantoh's point, I think `id` can be used here instead of `visualID`? `visualID` seems redundant.

Reply:
Oops, just saw your [explanation](https://github.com/Dynavity/dynavity/pull/3#discussion_r594043289). However, my concern is that when the user edits the text of the canvas element, wouldn't the `visualID` change as well? This might cause issues. In addition, not all canvas elements will have text.

Anyway you're right, either @wltan or I can change this behaviour when we work on implementing the various canvas elements.